lib/locust_api.py

```diff
@@ -10,44 +10,62 @@ class LocustAuthHandler:
     """ Contains functions needed to interact with the Locust API in an easy way
         for testing.
     """
-    LOCUST_USER = 'LOCUST_USER'
-    LOCUST_PASS = 'LOCUST_PASS'
+    LOCUST_USER_FF = 'LOCUST_USER'
+    LOCUST_PASS_FF = 'LOCUST_PASS'
+    LOCUST_USER_CGAP = 'LOCUST_USER_CGAP'
+    LOCUST_PASS_CGAP = 'LOCUST_PASS_CGAP'
 
-    def __init__(self, auth=None, use_env=True):
+    def __init__(self, auth=None, use_env=True, is_ff=True):
```

Comment:
I might prefer 
```
def __init__(self, auth=None, use_env=True, *, is_ff):
```
so that `is_ff` is a required keyword arg. I get that it presently defaults but having a default seems arbitrary and favors ff.